slack/greeting

```diff
@@ -0,0 +1,19 @@
+Hey {{Joiner}}, Welcome to the JSON Schema Slack Server!
+
+Whatever reason you chose to join us, we want you to know, you're welcome!
```

Comment:
"Whatever the reason"?  Maybe that's just an American thing?

Also "you're welcome" might be misinterpreted as a response to "thank you."  Maybe, "we're glad you're here" or "we're glad to have you" or something.